server/accounts.go

```diff
@@ -2889,6 +2885,21 @@ func (ur *URLAccResolver) Fetch(name string) (string, error) {
 	return string(body), nil
 }
 
+// Store will store or update the jwt claim of account `name`
+// to the base url, appending the account name onto the end.
+func (ur *URLAccResolver) Store(name, jwt string) error {
+	url := ur.url + name
+	resp, err := http.Post(url, "application/jwt", strings.NewReader(jwt))
```

Comment:
`ur` has a `c` http client that should be used here. So `ur.c.Post(` instead.

Reply:
Thanks, I updated like you suggested.